.github/dependabot.yml

```diff
@@ -0,0 +1,13 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
```

Comment:
hey I'm trying to use your PR to learn how this works. I can understand the stuff below (/tests picks up the mkdocs build requirements etc). But I'm unsure what this is doing. Is it a placeholder in case we add some requirements for github-actions, or is it picking up a file of requirements in the main dir that I'm just not recognizing?

Reply:
@samccann this keeps the actions up to date in the workflows. see here as an example: https://github.com/ansible/ecosystem-documentation/blob/f691e36bc45e99b2e3893d2b85ab0bd7a7e8e118/.github/workflows/reusable-nox.yml#L19

so when version 5 of the checkout action is available dependabot will automatically bump this and send us a PR https://github.com/marketplace/actions/checkout